script.js

```diff
@@ -0,0 +1,366 @@
+  formName.textContent = 'Add name of dish';
+  const nameInput = document.createElement('input');
+  nameInput.id = 'recipe-title';
+  formName.appendChild(nameInput);
+  newForm.appendChild(formName);
+
+  const formImg = document.createElement('label');
+  formImg.textContent = 'Add image of dish';
+  const imgInput = document.createElement('input');
+  imgInput.id = 'recipe-image';
+  formImg.appendChild(imgInput);
+  newForm.appendChild(formImg);
+}
+
+function createIngredientsSection(newForm) {
+  const labelIngredients = document.createElement('label');
+  labelIngredients.textContent = 'Ingredients (min 5):';
+  newForm.appendChild(labelIngredients);
+
+  const ingredientsContainer = document.createElement('div');
+  ingredientsContainer.id = 'ingredients-container';
+  newForm.appendChild(ingredientsContainer);
+
+  const addIngredientButton = document.createElement('button');
+  addIngredientButton.type = 'button';
+  addIngredientButton.textContent = 'Add ingredient';
+  addIngredientButton.id = 'ingredient-button';
+
+  addIngredientButton.addEventListener('click', () => {
+    createFormForIngredient(ingredientsContainer);
+  });
+
+  newForm.appendChild(addIngredientButton);
+
+  for (let i = 0; i < 5; i++) {
+    createFormForIngredient(ingredientsContainer);
+  }
+
+  return ingredientsContainer;
+}
+
+function createFormForIngredient(ingredientsContainer) {
+  const ingredientInput = document.createElement('div');
+  ingredientInput.classList.add('ingredient');
+
+  const nameInput = document.createElement('input');
+  nameInput.classList.add('ingredient-name');
+  nameInput.placeholder = 'Ingredient name';
+
+  const amountInput = document.createElement('input');
+  amountInput.classList.add('ingredient-amount');
+  amountInput.placeholder = 'Amount';
+
+  ingredientInput.appendChild(nameInput);
+  ingredientInput.appendChild(amountInput);
+  ingredientsContainer.appendChild(ingredientInput);
+}
+
+function createDescriptionField(newForm) {
+  const formDescription = document.createElement('label');
+  formDescription.textContent = 'Description:';
+  const descriptionInput = document.createElement('textarea');
+  descriptionInput.id = 'recipe-description';
+  formDescription.appendChild(descriptionInput);
+  newForm.appendChild(formDescription);
+}
+
+function createSubmitButton(newForm, ingredientsContainer) {
+  const addRecipe = document.createElement('button');
+  addRecipe.textContent = 'Add recipe';
+  addRecipe.id = 'add-recipe-button';
+  addRecipe.type = 'button';
+
+  addRecipe.addEventListener('click', function (event) {
+    event.preventDefault();
+
+    const title = document.querySelector('#recipe-title').value.trim();
+    const image = document.querySelector('#recipe-image').value.trim();
+    const description = document.querySelector('#recipe-description').value.trim();
+
+    if (!title || !image || !description) {
+      alert('Please fill in all required fields: title, image URL, and description.');
```

Comment:
Have you tried using the `required` property on your input elements? The browser already has these kinds of checks built in. You can then use `checkValidity` and `reportValidity` to check your form and to display a message to the user, as well as stopping the execution of your submit handler. See [Validating forms using JavaScript on MDN](https://developer.mozilla.org/en-US/docs/Learn_web_development/Extensions/Forms/Form_validation#validating_forms_using_javascript)